ui/src/virtualconsole/vcxypadproperties.cpp

```diff
@@ -510,6 +514,22 @@ void VCXYPadProperties::updatePresetsTree()
     m_presetsTree->blockSignals(false);
 }
 
+void VCXYPadProperties::selectItemOnPresetsTree(quint8 presetId)
+{
+    m_presetsTree->blockSignals(true);
+
+    for (int i = 0; i < m_presetsTree->topLevelItemCount(); ++i)
+    {
+        QTreeWidgetItem* treeItem = m_presetsTree->topLevelItem(i);
+        if (treeItem->data(0, Qt::UserRole).toUInt() == presetId)
+        {
+            treeItem->setSelected(true);
+            m_presetsTree->blockSignals(false);
+            return;
+        }
+    }
+}
```

Comment:
If the item is not found, the signals stay blocked forever. QSignalBlocker would be useful here, but it's a Qt5 class. It might be worth to backport it for our use until we drop Qt4 support.

Without QSignalBlocker, m_presetsTree->blockSignals(false) between lines 530 and 531 -OR- comment stating why it's not needed there.